### PR TITLE
More model cards

### PR DESCRIPTION
**Pull request by sgugger**

### CODE REVIEW

**Review by LysandreJik — APPROVED** [review]
Woa these look great! (please wait for Julien's approval before merge, I'm no expert when it comes to model cards)

**Comment by patrickvonplaten** [comment]
Looks great!

**Review by patrickvonplaten — APPROVED** [review]

File: model_cards/bert-base-cased-README.md [hunk 1]
```diff
@@ -1,10 +1,230 @@
 ---
+language: english
 tags:
 - exbert
-
 license: apache-2.0
+datasets:
+- bookcorpus
+- wikipedia
```
Comment (julien-c) [inline model_cards/bert-base-cased-README.md:8]: In cases where the exact pretraining code was not open-sourced (as is the case for BERT, if I'm not mistaken), I'm assuming it is still "scientifically correct" to link to the `huggingface/nlp` versions of the datasets.

Right @thomwolf?

**Comment by julien-c** [comment]
That's awesome, thanks!

File: model_cards/bert-base-cased-README.md [hunk 2]
```diff
@@ -1,10 +1,230 @@
...
 
+# BERT base model (cased)
+
+Pretrained model on English language using a masked language modeling (MLM) objective. It was introduced in
+[this paper](https://arxiv.org/abs/1810.04805) and first released in
+[this repository](https://github.com/google-research/bert). This model is case-sensitive: it makes a difference between
+english and English.
+
+Disclaimer: The team releasing BERT did not write a model card for this model so this model card has been written by
+the Hugging Face team.
```
Comment (julien-c) [inline model_cards/bert-base-cased-README.md:19]: Great addition

File: model_cards/bert-base-cased-README.md (hunk 1 above)
Comment (thomwolf) [reply model_cards/bert-base-cased-README.md:8]: Yes, I confirm. We are now hosting a version of the `bookcorpus` (thanks to @mariamabarham + community) which is awesome!

**Pull request merged**